README.md

```diff
@@ -72,24 +72,6 @@ Now you need to add the TypeScript language service plugin to your `tsconfig.jso
 }
 ```
 
-## Support for multiple errors
-
-If you expect more than one error, you can add multiple comments, like this:
-
-```ts
-type Person = {
-  readonly name: string;
-};
-
-const person: Person = {
-  name: "Bob",
-};
-
-// 💥 Expect error 2704: … cannot be a read-only property.
-// 💥 Expect error 2790: … must be optional.
-delete person.name;
-```
-
```

Comment:
@irenama I had to remove the support for multiple errors as the new TypeScript versions seems to only report one error at a time. So in this case it would first only report the first error. When that error got fixed, it reported the second one.

We need to check in our workshops where we specify more than one `Expect error`

Reply:
Thanks a bunch for seeing the PR through 🙂